fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/logging/LogMessageKeys.java

```diff
@@ -32,14 +32,28 @@
 public enum LogMessageKeys {
     // general keys
     TITLE("ttl"),
+    CLUSTER("cluster"),
     SUBSPACE("subspace"),
     SUBSPACE_KEY("subspace_key"),
+    INDEX_SUBSPACE("indexSubspace"),
     CALLING_CLASS("calling_class"),
     CALLING_METHOD("calling_method"),
     CALLING_LINE("calling_line"),
     FUTURE_COMPLETED("future_completed"),
     SCAN_PROPERTIES("scan_properties"),
     READ_VERSION("read_version"),
+    OLD("old"),
+    NEW("new"),
```

Comment:
I'm kind of on the fence about adding such constants that are used by test cases or, generally, unlikely to ever appear in other code.  Not sure if anyone else has a strong opinion of this.

Reply:
- These keys are used in 'fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBMetaDataStore.java'. The key names are not idea. This is not test code.
- It was decided to not standardize key names at this point.
- Standardizing at this point would be a trivial exercise though.

Reply:
It does seem a little weird to add test-case only keys (like, say, `TOTAL_RESULT_COUNT`). In theory, those could go in their own enum? `TestLogMessageKeys`?

Reply:
DONE:
- Moved all test-only keys to TestLogMessageKeys.